api/client/build.go

```diff
@@ -176,9 +167,22 @@ func (cli *DockerCli) CmdBuild(args ...string) error {
 		return err
 	}
 
-	// Wrap the tar archive to replace the Dockerfile entry with the rewritten
-	// Dockerfile which uses trusted pulls.
-	context = replaceDockerfileTarWrapper(context, newDockerfile, relDockerfile)
+	var resolvedTags []*resolvedTag
+	if os.Getenv("DOCKER_CONTENT_TRUST") != "" {
```

Comment:
There is another way of enabling DCT, by providing  `--disable-content-trust=false`